### PR TITLE
Feat/add cargo ledger

### PR DESCRIPTION
**Pull request by yogh333**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -22,15 +22,31 @@ jobs:
...
         with:
           command: clippy
-          args: --target ${{ matrix.target }}
+          args: -p ${{ matrix.package }} --target ${{ matrix.target }}
   
+  clippy-cargo-ledger:
+    name: Run static analysis for cargo-ledger
+    runs-on: ubuntu-latest
+    container:
+        image: ghcr.io/ledgerhq/ledger-app-builder/ledger-app-dev-tools:latest
```
Comment (agrojean-ledger) [inline .github/workflows/ci.yml:39]: Not sure we need to be in the dev-tools container for cargo-ledger jobs... :thinking:

Comment (yogh333) [reply .github/workflows/ci.yml:39]: Well, the idea is to be consistent with regards to other CI tests: all are performed within dev-tools container, so I have decided to keep the same test environment. For cargo-ledger, we are using a stable toolchain for cargo-ledger, so there's should not be any diff even if we were using an ubuntu-based only container.

File: cargo-ledger/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[package]
+name = "cargo-ledger"
+version = "1.2.0"
```
Comment (agrojean-ledger) [inline cargo-ledger/Cargo.toml:3]: Bump the version ?

Comment (agrojean-ledger) [reply cargo-ledger/Cargo.toml:3]: Then we should update the version in the app-builder docker images

Comment (yogh333) [reply cargo-ledger/Cargo.toml:3]: OK I will bump the version to 1.2.1

Comment (agrojean-ledger) [reply cargo-ledger/Cargo.toml:3]: I will update the version in the app builder image in https://github.com/LedgerHQ/ledger-app-builder/pull/91

**Pull request merged**